### PR TITLE
Added 4 Modern Themes That Use CSS To Make A Gradient Background

### PR DESCRIPTION
**Pull request by outercloudstudio**

### CODE REVIEW

**Comment by outercloudstudio** [comment]
I removed the end and nether themes because they were not good enough.

**Comment by outercloudstudio** [comment]
Also I added 4 modern themes that I think look really cool.

File: plugins/ModernTheme/manifest.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"author": "Outer Cloud Studio",
+	"id": "3a87aa2a-d494-4c93-ae28-6352928e2b7b",
+	"version": "1.0.0",
+	"name": "Modern Themes",
+	"description": "Adds multiple asthetic modern themes to select in File > Prefrences > Setting > Apearence!",
```
Comment (solvedDev) [inline plugins/ModernTheme/manifest.json:6]: Typo: "Prefrences"

**Review by solvedDev — CHANGES_REQUESTED** [review]

**Comment by outercloudstudio** [comment]
The typo should be fixed now.

**Review by solvedDev — APPROVED** [review]

**Pull request merged**